blocks/editable/index.js

```diff
@@ -825,7 +827,7 @@ export default class Editable extends Component {
 			formatters,
 		} = this.props;
 
-		const ariaProps = pickAriaProps( this.props );
+		const ariaProps = { ...pickAriaProps( this.props ), 'aria-multiline': MULTI_LINE_TAGS.indexOf( Tagname ) > -1 };
```

Comment:
Should `aria-multiline` be dictated by `this.props.multiline` ?

Reply:
`this.props.multiline` is undefined for a ParagraphBlock and is set to `li` for a ListBlock.  I looked at this prop first but it wasn't suitable for this purpose...

Reply:
Okay. I suspect future maintainers may have a similar reaction to mine in not understanding the difference between how `this.props.multiline` and `aria-multiline` are handled. Perhaps we could do for a code comment or documentation clarifying this.

Reply:
I think the best approach would to re-name the multiline prop to something more appropriate, maybe `multilineTag` as this is how its used...